tests/impact/client/test_client.py

```diff
@@ -294,3 +294,18 @@ def test_get_executions_for_workspace(executions, model_compile, experiment_exec
     opeartions = list(client.get_executions(IDs.WORKSPACE_SECONDARY))
     assert len(opeartions) == 1
     assert isinstance(opeartions[0], ExperimentOperation)
+
+
+def test_get_me(user):
```

Comment:
I did try a VCR implementation for this by setting the "scrubbed" response_body to the dummy dict used in this fixture in this MR but the tests would fail during the regeneration as the responses are only scrubbed before saving to disk, so while generating the cassette it would try to match with actual response body sent by the server. IAlso it seemed like an overkill for this one specific test.